File: Beecrowd/1183/1183.c
/* Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1183 */
#include <stdio.h>

int main(){
        char Op;
        float M[12][12];
        float sum = 0, num = 0;
        scanf("%c", &Op);
        for(int i = 0; i < 12; i++){
                for(int j = 0; j < 12; j++){
                        scanf("%f", & M[i][j]);
                        if(i < j){
                                sum += M[i][j];
                                num+=1;
                        }
                }
        }
        float avg = sum / num;
        if(Op == 'S'){ printf("%.1f\n", sum); }
        else{ printf("%.1f\n", avg); }

        return 0;
}

File: Beecrowd/1183/1183.py
# Problem statement: https://www.beecrowd.com.br/judge/en/problems/view/1183
Op = input()
sum = 0
num = 0
for i in range(0,12):
    for j in range(0,12):
        N = float(input())
        if i < j:
            sum += N
            num += 1
if Op == 'S':
    print(sum)
else:
    avg = sum / float(num)
    print("%.1f" % avg)

